src/Serilog.Sinks.Elasticsearch/Serilog.Sinks.ElasticSearch.nuspec

```diff
@@ -11,12 +11,12 @@
     <iconUrl>http://serilog.net/images/serilog-sink-nuget.png</iconUrl>
     <tags>serilog logging elasticsearch</tags>
     <dependencies>
-      <dependency id="Serilog" version="[1.4.204,2)" />
-      <dependency id="Elasticsearch.Net" version="1.1.2" />
+      <dependency id="Serilog" version="[1.5.14,2)" />
+      <dependency id="Elasticsearch.Net" version="1.7.1" />
     </dependencies>
   </metadata>
   <files>
-    <file src="bin40\Release\Serilog.Sinks.Elasticsearch.dll" target="lib\net40" />
-    <file src="bin40\Release\Serilog.Sinks.Elasticsearch.xml" target="lib\net40" />
+    <file src="bin\Release\Serilog.Sinks.Elasticsearch.dll" target="lib\net45" />
```

Comment:
Dropping support for .NET 4? (The 4.5 binaries are included in the package by default.)